python_modules/dagster/dagster/__init__.py

```diff
@@ -205,6 +206,10 @@
     Definitions as Definitions,
     create_repository_using_definitions_args as create_repository_using_definitions_args,
 )
+from dagster._core.definitions.definitions_loader import (
+    DefinitionsLoadContext as DefinitionsLoadContext,
+    DefinitionsLoader as DefinitionsLoader,
+)
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +185 to +212]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Let's not expose these publicly yet until we have docs and a rollout plan<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24337?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
@PedramNavid as you developing the playbook for public APIs I think when to do top-level exports should be in that guidance.